src/transformers/trainer.py

```diff
@@ -1079,20 +1079,18 @@ def train(
             num_update_steps_per_epoch = max_steps
 
         delay_optimizer_creation = self.sharded_ddp is not None and self.sharded_ddp != ShardedDDPOption.SIMPLE
-        model = self.model
-        if self.args.ort:
-            from onnxruntime.training import ORTModule
+        if args.ort:
+            from onnxruntime.training.ortmodule import ORTModule
             logger.info("Converting to ORTModule ....")
             model = ORTModule(self.model)
             self.model_wrapped = model
-        if self.args.deepspeed:
-            if self.args.ort:
-                self.model = model
         if args.deepspeed:
+            if args.ort:
+                self.model = model
```

Comment:
Is this line necessary if self.model will be assigned to deepspeed_engine.module._original_module if args.ort in line 1093?

Reply:
Deepspeed wraps self.model and we want deepspeed to use model wrapped by ORTModule.